src/data/ab-attrs/post-attack-ab-attr.ts

```diff
@@ -1,56 +1,50 @@
-import type { PokemonAttackCondition } from "#app/@types/PokemonAttackCondition";
 import type { Move } from "#app/data/move";
 import { MoveCategory } from "#enums/move-category";
 import type { Pokemon } from "#app/field/pokemon";
-import type { HitResult } from "#enums/hit-result";
 import { AbAttr } from "./ab-attr";
 
 export class PostAttackAbAttr extends AbAttr {
-  private readonly attackCondition: PokemonAttackCondition;
+  /** Does this effect only apply to attack moves? */
+  private readonly attackOnly: boolean;
```

Comment:
Is this a question from you or the "explanation"?

```suggestion
  /** Whether it only applies to attack moves. */
  private readonly attackMovesOnly: boolean;
```